script-library/go-debian.sh

```diff
@@ -70,11 +70,20 @@ if [ "${TARGET_GO_VERSION}" = "latest" ] ||  [ "${TARGET_GO_VERSION}" = "current
     TARGET_GO_VERSION=$(curl -sSL "https://golang.org/VERSION?m=text" | sed -n '/^go/s///p' )
 fi
 
+ARCHITECTURE="$(uname -m)"
+case $ARCHITECTURE in
+    x86_64) ARCHITECTURE="amd64";;
+    aarch64 | armv8*) ARCHITECTURE="arm64";;
+    aarch32 | armv7* | armvhf*) ARCHITECTURE="armv6l";;
+    i?86) ARCHITECTURE="386";;
+    *) echo "(!) Architecture unsupported"; exit 1 ;;
```

Comment:
Maybe print the `$ARCHITECTURE` in the error `echo` to make it easier for someone (probably us!) to debug if this is reported

Reply:
Good idea.